crates/autopilot/src/domain/auction/order.rs

```diff
@@ -25,6 +29,7 @@ pub struct Order {
     pub buy_token_balance: BuyTokenDestination,
     pub app_data: AppDataHash,
     pub signature: Signature,
+    pub quote: Option<domain::Quote>,
```

Comment:
This should probably be non-optional in autopilot domain

Reply:
According to the PR description, that is the plan:
> After 1-2 releases, the quote field from the fee will be dropped and the introduced order's quote will be used instead.

> This depends if [these values](https://github.com/cowprotocol/services/pull/2854/files#diff-4e1d38ff613813c27413004d11bacde9bf40357c838ceaed5abaf51d904fc076R121-R126) should be used for protocol fees only.

I was hoping we could use them. Will decide in a separate PR.